browser/ui/webui/private_new_tab_page/brave_private_new_tab_page_handler.cc

```diff
@@ -93,6 +94,20 @@ void BravePrivateNewTabPageHandler::GetIsTorConnected(
   std::move(callback).Run(is_connected);
 }
 
+void BravePrivateNewTabPageHandler::GetIsTorDisabled(
+    GetIsTorDisabledCallback callback) {
+  bool is_disabled = false;
```

Comment:
Q. I'm curious why `is_disabled`'s default value is `false`.
Is this to make it enabled even when `profile` doesn't exist?

Reply:
Yes correct, but it's going to change with the below change.